_includes/gallery-template.html

```diff
@@ -1,51 +0,0 @@
-<div id="user-gallery" class="banner banner--hr js-dynamic-section" data-group="Contributing" data-label="User Gallery">
-  <div class="constrained constrained--med">
-    <div class="padded-med">
-      <h2>User Gallery</h2>
-      <table class="compact">
-        {% for model in site.data.gallery %}
```

Comment:
Where does this data come from?  Is there more that needs to be removed?

Reply:
I added a commit that removes `_data`.